test/utxo-select.test.js

```diff
@@ -725,25 +728,25 @@ describe('UTXO Select', () => {
     );
     deepStrictEqual(t(est, est.smallest, true, false), {
       amounts: [1n, 2n, 4n, 8n, 16n, 32n, 64n, 128n, 256n, 512n, 1024n],
-      change: 261n,
-      expFee: 1686n,
+      change: 106n,
+      expFee: 1685n,
```

Comment:
change explained by fix on L683

`261 - ( 256 - 100) = 105`

I'm not sure where the 1n fee difference comes from.